content/sensu-go/6.8/catalog/build-private-catalog.md

```diff
@@ -0,0 +1,197 @@
+  - filters:
+    - entity.system.os == 'linux'
+    - entity.system.arch == 'arm'
+    - entity.system.arm_version == 7
+    headers: null
+    sha512: 714e777c214fd5a7210b67030eb761f5d8c7f8e9ba55f6a0d64872f43f27848eaf51c17bd7b3e3efbdc419d4e4754c6143c705b06ddd750009f8068872e5d35d
+    url: https://assets.bonsai.sensu.io/02bff14ff7f692daab5cace39dcc6e184751285a/nginx-check_0.1.0_linux_armv7.tar.gz
+  - filters: ...
+{{< /code >}}
+
+If needed, replace `assets.bonsai.sensu.io` with your preferred URL in asset `builds.url` values in all `sensu-resources.yaml` files before you continue.
+
+## Install the catalog-api command line tool
+
+The Sensu Catalog API includes the catalog-api command line tool.
+The catalog-api tool is an open-source static API generator: it renders static HTTP API content that the Sensu web UI can consume.
+
+To install the catalog-api tool:
+
+1. Clone the Sensu Catalog API repository:
+{{< code shell >}}
+git clone https://github.com/sensu/catalog-api
+{{< /code >}}
+
+2. Navigate to the local catalog-api repository:
+{{< code shell >}}
+cd catalog-api
+{{< /code >}}
+
+3. Build the `catalog-api` tool:
+{{< code shell >}}
+go build
+{{< /code >}}
+
+## Clone and validate the integration repository
+
+The catalog-api tool consumes content from a repository that includes all the files required to build a catalog of integrations.
+Follow these steps to clone your repository and validate that all files are organized properly:
+
+1. Clone the repository that stores your Sensu integrations.
+This example uses Sensu's public integration repository:
+{{< code shell >}}
+git clone https://github.com/sensu/catalog
+{{< /code >}}
+
+2. Navigate to your local copy of the repository that stores the Sensu integrations.
+This example uses https://github.com/sensu/catalog, so the repository is `catalog`: 
+{{< code shell >}}
+cd ../catalog
+{{< /code >}}
+
+3. Validate the integration repository contents:
+{{< code shell >}}
+../catalog-api/catalog-api catalog validate --repo-dir .
+{{< /code >}}
+
+## Generate the private catalog
+
+With a validated repository, you can generate your private catalog locally:
+
+{{< code shell >}}
+../catalog-api/catalog-api catalog generate
+{{< /code >}}
+
+The `generate` subcommand generates the static API in a temporary directory, `/tmp/generated-api/`.
+
+These catalog builds are versioned so that every previous iteration is available.
+
+The catalog-api tool generates builds into a checksum-based output directory structure.
+The path to the latest or production Catalog API content is managed by a `version.json` file (in this example, the file would be https://catalog.sensu.io/version.json) that instructs the Sensu web UI to load API contents from the specified checksum subdirectory.
+For example:
+
+{{< code text >}}
+{
+  "release_sha256": "d0e2ba810c3d546c82121406fb5f214e66aeb7fe9026706f9a7391463cf4da19",
+  "last_updated": 1657574563
+}
+{{< /code >}}
```

Comment:
I think I'm missing something here about what users need to know and do with versions. Maybe also something to do with git tags?

Reply:
Note from Jef's customer demo:

catalog-api tool uses the version tags to create versions of integrations and present them to users within the catalog. You are not limited to providing only the most recent version; you can provide older versions as a fallback.

Reply:
Also in Jef's customer demo, he says that you have to create git tags and uses this command:

```
git tag <integration_namespace>/<integration_filename>/<YYYYMMDD>.0.0
```

Reply:
This is correct. Do you have any additional questions about this process?